### ISSUE TITLE
A nonnullable DataProperty cannot have a null defaultValue

### ISSUE THREAD

**Issue by ganySA** [Breeze/breeze-client#50]
Hi

I seem to have this after the latest version.. It does it on every single property of my medadata.

Seems to be coming from here - my metadata has not changed.
```
                else {
                    this.defaultValue = this.dataType.defaultValue;
                    const msg = "A nonnullable DataProperty cannot have a null defaultValue. Name: " + (this.name || this.nameOnServer);
                    if (this.defaultValue === null) {
                        throw new Error(msg); // if defaultValue is explicity set to null, that's an error
                    }
                    else {
                        console.warn(msg); // if defaultValue is not set, that's a warning
                    }
                }
```

**Comment by ganySA** [comment]
A further comment my metaData does not contain defaultValue - i am using DBFirst with context generator for my classes.

**Comment by steveschmitt** [comment]
Ugh, sorry!  I overlooked that somehow in my testing.  Thanks for letting me know right away.

**Comment by ganySA** [comment]
Any chance you can publish to npm?

**Comment by steveschmitt** [comment]
Fixed in release 2.1.1.
